network-api/networkapi/templates/fragments/blocks/nav/button.html

```diff
@@ -0,0 +1,13 @@
+{% load static wagtailadmin_tags %}
+
+{% if style == "primary" %}
+    {% fragment as nav_button_style %}tw-btn-primary tw-py-4 tw-px-8{% endfragment %}
+{% elif style == "primary-full-width" %}
+    {% fragment as nav_button_style %}tw-btn-primary tw-min-w-full tw-py-4 tw-px-8{% endfragment %}
+{% elif style == "cta" %}
+    {% fragment as nav_button_style %}tw-leading-[130%] tw-text-blue-80 tw-p-2 hover:tw-underline{% endfragment %}
+{% endif %}
+
+{% fragment as base_styles %}link-button tw-font-bold tw-text-lg{% endfragment %}
+
+<a class="{{ nav_button_style }} {{ base_styles }} after:tw-content-['_→'] after:tw-font-['Nunito_Sans']" href="{{ value.url }}" {% if value.open_in_new_window %}target="_blank"{% endif %}>{{ value.label }}</a>
```

Comment:
Let's use our custom tailwind font variable (tw-font-sans) instead of tw-font-['Nunito_Sans']

Reply:
This one needs to be only `['Nunito Sans']`, otherwise the icons get messed up. See https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12146#discussion_r1556373554